ember/app/ui/components/dynamic-table/template.hbs

```diff
@@ -16,7 +16,12 @@
   </thead>
   <tbody class="uk-overflow-auto">
     {{#each @data as |entry|}}
-      <tr class="uk-table-middle">
+      {{! template-lint-disable no-invalid-interactive }}
```

Comment:
`role` attribute is set through a condition. ember-template-lint does not understand that.